.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: "Stale issues and pull requests"
+
+on:
+  schedule:
+  - cron: "30 17 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          operations-per-run: 100
+          days-before-stale: 60
+          days-before-close: 30
+          exempt-issue-labels: 'needs-triage'
+          exempt-pr-labels: 'needs-triage'
```

Comment:
I think we should exclude issues labeled as bug, product issue. We can mark the ones which are waiting for customer response as stale due to inactivity. Thoughts?